shared/chat/conversation/list-area/normal/index.desktop.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+/* eslint-env browser */
```

Comment:
why is this necessary?

Reply:
eslint was marking `MouseEvent` as undefined

Reply:
you're using the wrong type: https://github.com/facebook/flow/blob/v0.76.0/lib/react-dom.js#L193

Reply:
oh it's actually complaining about `HTMLInputElement` and `HTMLTextAreaElement` as well